docs/topics/mta-web-subv-rep.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-subv-rep_{context}"]
+= Configuring Subversion repositories
+
```

Comment:
You need an introductory sentence, even if it's "You can do x in the y view"

Reply:
You can configure Subversion repositories with the following steps.